roles/neutron-control/tasks/main.yml

```diff
@@ -27,7 +27,7 @@
   tags: db-migrate
 
 - name: sync neutron database for lbaas
-  command: neutron-db-manage --service lbaas --config-file /etc/neutron/neutron.conf \
+  command: neutron-db-manage --subproject neutron-lbaas --config-file /etc/neutron/neutron.conf \
```

Comment:
Well, I rolled back the file to use "--service lbaas" and tried to run again, and found this issue is gone.. So weird...
Not sure what happened, but anyway, I will continue to use "--service lbaas".